performance_manager/pyproject.toml

```diff
@@ -38,8 +38,10 @@ disable = [
   "broad-except",
   # caught by black
   "line-too-long",
+  # we're logging everything so its "ok"
+  "lost-exception"
 ]
-good-names = ["e", "i"]
+good-names = ["e", "i", "s"]
```

Comment:
i added this so that we can get by `with session.begin() as s:` but think we should probably remove it when we move a bunch of stuff to postgres_utils.py.